wp-content/themes/goonj-crm/validation.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const form = document.querySelector("form");
+  const emailInput = document.getElementById("email");
+  const phoneInput = document.getElementById("phone");
+
+  form.addEventListener("submit", function (event) {
+      let valid = true;
+
+      document.querySelectorAll(".errorMessage").forEach((error) => error.remove());
+
+      // Validate email
+      const emailValue = emailInput.value;
+      if (!/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(emailValue)) {
+          valid = false;
+          showError(emailInput, "The e-mail address entered is invalid.");
+      }
+
+      // Phone validation code commented out for now. Uncomment when needed.
+      /*
+      const phoneValue = phoneInput.value;
+      if (!/^\d{10}$/.test(phoneValue)) {
+          valid = false;
+          showError(phoneInput, "Phone number must be exactly 10 digits.");
+      }
+      */
```

Comment:
The phone validation code is currently commented out to avoid affecting existing users logging into the collection camp. We will uncomment this code later when it is needed.